### PR TITLE
refactor fetch exchange data

### PR DESCRIPTION
**Pull request by secmask**
- refactor fetch exchange data in the way easier to limit number of concurrent request
- move price(orderbook data) to separate table, so we can partition it daily and clean up old data.
- fetch_data table no longer compress data, read operation will detect and decompress old data on needed.

### CODE REVIEW

File: exchange/binance.go [hunk 1]
```diff
@@ -180,71 +175,62 @@ func (bn *Binance) FetchOnePairData(
...
-		exPrice, ok := value.(common.ExchangePrice)
-		if !ok {
-			err = fmt.Errorf("value (%v) cannot be asserted to ExchangePrice", value)
-			return false
-		}
-		result[tokenPairID] = exPrice
-		return true
-	})
+	close(jobs)
+	concurrentFactor := 8
```
Comment (halink0803) [inline exchange/binance.go:220]: this is fixed number, should it be a const?

**Review by halink0803 — APPROVED** [review]
LGTM. One question still.

File: exchange/binance.go (hunk 1 above)
Comment (secmask) [reply exchange/binance.go]: okay, I make it const

**Review by halink0803 — APPROVED** [review]
LGTM

**Pull request merged**